### DISCUSSION TITLE
Please convert the python example to scheme

### DISCUSSION THREAD

**Discussion by rolly-ng** [NanoComp/meep#1967] [Q&A] [answered]
Hi, I read about the example in the thermal example in [http://www.simpetus.com/projects.html#meep_thermal_radiation](url)

I am new to the scheme language and I would like to convert the example script to scheme. Can anyone help? Thanks!

I would like to know how to do the python main(args) in scheme? so, that I can call the meep binary to process the arguments as the example?

`mpirun -np ${np} meep emitter.ctl -empty -aa ${a} > flux0_a${a}.out;`

Where can I find similar example with scheme language?

Thank you,
rolly

**Comment by oskooi** [comment, marked as answer]
The Scheme version of this example is already available: http://simpetus.com/projects_scheme.html#meep_thermal_radiation.

**Comment by stevengj** [comment]
Note that most future development of Meep will be for the Python front-end, so I would encourage you to use Python rather than Scheme with Meep if possible.

**Reply by rolly-ng** [reply]
Thank you but I suppose using the meep binary shall enhance the speed on multinode cluster? I have just completed the example but the result is much different from the webpage, I have increased the number of incident angles and repeating the run.
![a4 3_r1 72](https://user-images.githubusercontent.com/24451125/155950812-d1efc649-bbf9-4dc1-91c9-4029a8cfdd72.png)
